envied.gemspec

```diff
@@ -23,7 +23,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency "coercible", "~> 1.0"
   spec.add_dependency "thor", "~> 0.15"
 
-  spec.add_development_dependency "bundler"
-  spec.add_development_dependency "rake"
```

Comment:
No problem, I'll make that change. Although its changed slightly with a new gem which makes sense since Bundler 2 is out now.

```ruby
  spec.add_development_dependency "bundler", "~> 2.0"
  spec.add_development_dependency "rake", "~> 10.0"
```

Reply:
Interesting that the default still outputs `~> 10.0` for `rake` since it's so old. I'll update that to be current with `~> 12.0` instead.